pkg/version/version.go

```diff
@@ -19,12 +19,13 @@ package version
 import "fmt"
 
 // The below variables will be overrriden using ldflags set by goreleaser during the build process
-var VERSION = "DEV"
-var GIT_COMMIT = "NONE"
-var BUILD_DATE = "UNKNOWN"
+
+var Version = "DEV"
+var GitCommit = "NONE"
+var BuildDate = "UNKNOWN"
```

Comment:
This would break the release. We have to update ldflags in `.goreleaser` as per the changes. https://github.com/kanisterio/kanister/blob/master/.goreleaser.yml#L15

Reply:
Oh, right. My bad